Facades/Notifications/Notifications.php

```diff
@@ -175,7 +181,8 @@ public function onNotify($transactionID, $paymentID)
             }
         }
 
+        $this->orderUpdater->updateOrderHistoryComment($order);
```

Comment:
doesnt this this need to be somewhere deeper integrated?
if not, then i think its missing in the "returnAction"....there are also status updates when the user comes back
so 1x return + 1x notification .......and probably in extjs? just thinking about what happens in shipping actions there

Reply:
this was indeed a bug,
and i also found another bug, that a non-mollie order history has been marked as mollie if the webhook is called again.
so there was no check, if a new entry was even made by mollie.

i havent heard back from you in this PR, so i fixed it myself and will decline this one